app/javascript/dashboard/store/modules/conversations/index.js

```diff
@@ -151,7 +151,7 @@ export const mutations = {
       selectedChatId: conversationId,
     });
 
-    if (!chat) return;
+    if (!chat?.attachments) return;
```

Comment:
Can you identify the case where the chat is initialized without attachments?

Reply:
I do have this same issue where chat is initialized without attachments, mainly with inline images.. will test this and give feedback aswell

Reply:
Thanks! Appreciate it.

Reply:
Doesn't work in my use case. Inline images still not attached.